app/javascript/tests/patient/Patient.test.js

```diff
@@ -72,9 +72,9 @@ describe('Patient', () => {
   });
 
   it('Properly renders identification section', () => {
-    const wrapper = shallow(<Patient details={mockPatient1} collapse={true} edit_mode={false} current_user={mockUser1} jurisdiction_paths={mockJurisdictionPaths} other_household_members={[]} can_modify_subject_status={true} workflow="global" />);
+    const wrapper = shallow(<Patient details={mockPatient1} collapse={true} edit_mode={false} current_user={mockUser1} jurisdiction_paths={mockJurisdictionPaths} other_household_members={[]} can_modify_subject_status={true} workflow="global" headingLevel={2} />);
     const section = wrapper.find('#identification');
-    expect(section.find('h4').text()).toEqual('Identification');
+    expect(section.find('Heading').children().at(0).text()).toEqual('Identification');
```

Comment:
is the `at(0)` necessary?

Reply:
same for the others

Reply:
In this case, yes because `Heading` is a new react component, and we are shallow rendering it. If you want me to remove the .at(0) I can totally change this to a `mount` instead of `shallow`

Reply:
update: you can remove the at(0) for almost all of these